Configuration.xcconfig

```diff
@@ -1,17 +1,17 @@
 RELAY_HOST = relay.walletconnect.com
 
 // Uncomment next line and paste your project id. Get this on: https://cloud.walletconnect.com/sign-in
-// PROJECT_ID = YOUR_PROJECT_ID
+PROJECT_ID = 9bfe94c9cbf74aaa0597094ef561f703
```

Comment:
⚠️

Reply:
We might utilise the https://danger.systems/swift/ to detect these automatically